packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './Image.module.css'
+
+type AspectRatio = '1:1' | '16:9' | '16:10' | '4:3' | 'custom'
+
+export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  ref?: React.Ref<HTMLImageElement>
```

Comment:
Could we please use BaseProps here with a forwardRef? Should be examples in other components.

Reply:
BaseProps gives issues related to ref className and id because they are duplicated. I would need to Omit them from the orignal class. I think instead we should remove BaseProps or use it for something non standard in general HTML interfaces.

Reply:
> they are duplicated and extending different types

BaseProps accept a generic type, where you can pass in the HTMLImageElement. As it's an intersection, it should combine any duplicates too. Using BaseProps is preferred because it's also future proofing it, where we can easily deploy global props in one-place, which we're thinking about doing for spacing.